CardScan/Classes/CreditCardUtils.swift

```diff
@@ -262,6 +264,57 @@ public struct CreditCardUtils {
         }
     }
     
+    /**
+        Returns the card's type (debit, credit, preiad, unknown) based on the card number
+        -   Parameter cardNumber: The card number as a string
+        -   Returns: The card's type as a CardType enum
+     */
+    public static func determineCardType(cardNumber: String) -> CardType {
+        guard let iin = Int(cardNumber.prefix(6)) else {
+            return .UNKNOWN
+        }
+        
+        let cardTypes: [(ClosedRange<Int>, CardType)]
+        if let cardTypeMap = self.cardTypeMap {
+            cardTypes = cardTypeMap
+        } else {
+            guard let filePath = Bundle.allBundles.compactMap({ $0.path(forResource: "card_types", ofType: "txt", inDirectory: "Config") }).first else {
```

Comment:
We usually use our resource bundle with something that looks like this:
```
guard let bundle = CSBundle.bundle() else { return }
let url = bundle.url(forResource: forResource, withExtension: withExtension)
```

But I'm open to this if you have a reason for using it instead

Reply:
This is the code I was struggling with for a while. You've just described exactly what I was missing, and I'll use that!